modules/creating-single-etcd-backup.adoc

```diff
@@ -0,0 +1,221 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-single-etcd-backup_{context}"]
+== Creating a single etcd backup
+
+Follow these steps to create a single etcd backup by creating and applying a custom resource (CR).
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have access to the OpenShift CLI (`oc`).
+
+.Procedure
+
+* If dynamically-provisioned storage is available, complete the following steps to create a single automated etcd backup:
++
+.. Create a persistent volume claim (PVC) named `etcd-backup-pvc.yaml` with contents such as the following example:
++
+[source,yaml]
+----
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: etcd-backup-pvc
+  namespace: openshift-etcd
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 200Gi <1>
+  storageClassName: etcd-backup-local-storage <2>
+  volumeMode: Filesystem
+----
+<1> The amount of storage available to the PVC. Adjust this value for your requirements.
+<2> The name of the `StorageClass` required by the claim. Adjust this value according to your environment.
++
+[NOTE]
+====
+Each of the following providers require changes to the `accessModes` and `storageClassName` keys:
+
+[cols="1,1,1"]
+|===
+|Provider|`accessModes` value|`storageClassName` value
+
+|AWS with the `versioned-installer-efc_operator-ci` profile
+|`- ReadWriteMany`
+|`efs-sc`
+
+|Google Cloud Platform
+|`- ReadWriteMany`
+|`filestore-csi`
+
+|Microsoft Azure
+|`- ReadWriteMany`
+|`azurefile-csi`
+|===
+====
++
+.. Apply the PVC by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f etcd-backup-pvc.yaml
+----
++
+.. Verify the creation of the PVC by running the following command:
++
+[source,terminal]
+----
+$ oc get pvc
+----
++
+.Example output
+[source,terminal]
+----
+NAME              STATUS    VOLUME   CAPACITY   ACCESS MODES   STORAGECLASS                AGE
+etcd-backup-pvc   Pending                                      etcd-backup-local-storage   51s
```

Comment:
I think this is case because you used a storage class that does not exist on the cluster you used :) 

If you remove the field `storageClassName: etcd-backup-local-storage` from the PVC, the status should be `Available` till they are being used by a Pod, then it should be `bound`

Reply:
I updated the status to `Bound`.